site/search/Search.scss

```diff
@@ -241,6 +241,20 @@ $reset-button-margin: 16px;
     }
 }
 
+.search-results__page-hit-container {
+    display: flex;
+}
+
+.search-results__page-hit-img-container {
+    margin-right: 16px;
+    @include sm-only {
+        display: none;
+    }
+}
+.search-results__page-hit-img {
+    max-width: 100px;
```

Comment:
I think this would work better with a min-width of also 100px.
That would make it so the img-loading related grid changes are less stark, and also that, if an image is missing for some reason, the grid isn't misaligned like so.
![CleanShot 2024-04-17 at 22 38 43@2x](https://github.com/owid/owid-grapher/assets/2641501/74aa2682-ed27-493a-991f-faeda071a62e)